resources/views/profile.blade.php

```diff
@@ -7,7 +7,7 @@
 
     <div class="py-12">
         <div class="max-w-7xl mx-auto sm:px-6 lg:px-8 space-y-6">
-            <div class="p-4 sm:p-8 bg-white dark:bg-gray-800 shadow sm:rounded-lg">
+            <div class="p-4 sm:p-8 bg-white dark:active:bg-gray-30000 shadow sm:rounded-lg">
```

Comment:
bg-gray-30000 doesn't exist as a CSS class, bg-gray-300 does though 😄 

Once you've changed this, you'll need to tell your application to rebuild your CSS. For this it uses a tool called [Vite](https://vitejs.dev) (pronounced _Veet_). To rebuild, run:

```
vite build
```

In your terminal